ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+TabWebNavigationDelegate.swift

```diff
@@ -222,6 +222,20 @@ extension BrowserViewController: TabWebNavigationDelegate {
       return true
     }
 
+    if let url = error.userInfo[NSURLErrorFailingURLErrorKey] as? URL {
+      // Check for invalid upgrade to https
+      if url.scheme == "https",  // verify failing url was https
```

Comment:
we have a `isSecureWebPage` which checks scheme contains "https". if here needs an explicit check, ignore my comment

Reply:
Also just code that moved, but yeah I can switch it in a future PR